src/components/ContactForm.tsx

```diff
@@ -0,0 +1,355 @@
+  position: 'absolute',
+  bottom: -1,
+  left: 0
+})
+
+const StyledError = styled(Typography)({
+  fontFamily: theme.typography.fontFamily,
+  fontSize: '0.8vw',
+  fontWeight: 'normal',
+  fontStretch: "normal",
+  fontStyle: "normal",
+  lineHeight: 'normal',
+  letterSpacing: 'normal',
+  color: 'red',
+  position: 'absolute',
+  bottom: 0,
+  left: '3vw'
+});
+
+const useSuccessSnackBarStyle = makeStyles({
+  root: {
+    backgroundColor: theme.palette.primary.main,
+    position: 'relative',
+    right: '17.5vw',
+    bottom: '7vw'
+  },
+  message: {
+    paddingLeft: '4vw',
+    fontFamily: theme.typography.fontFamily,
+    fontSize: '1vw',
+    fontWeight: 600,
+    fontStretch: "normal",
+    fontStyle: "normal",
+    letterSpacing: '-0.45px',
+    color: theme.palette.text.secondary,
+  },
+});
+
+const useFailsureSnackBarStyle = makeStyles({
+  root: {
+    backgroundColor: '#FF1744',
+    position: 'relative',
+    right: '17.5vw',
+    bottom: '7vw'
+  },
+  message: {
+    fontFamily: theme.typography.fontFamily,
+    fontSize: '1vw',
+    fontWeight: 600,
+    fontStretch: "normal",
+    fontStyle: "normal",
+    letterSpacing: '-0.45px',
+    color: theme.palette.text.primary,
+  },
+});
+
+
+interface Props {
+  classes?: string;
+}
+
+interface IFormInput {
+  name: string;
+  email: string;
+  message: string;
+}
+
+export const ContactForm: React.FC<Props> = (props: Props) => {
+
+  // FORM HOOK
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, errors, handleSubmit } = useForm<IFormInput>();
+
+  // INPUT STATE HOOKS
+  const [name, setName] = React.useState('');
+  const handleNameChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setName(event.target.value);
+  };
+  const [email, setEmail] = React.useState('');
+  const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value);
+  };
+  const [message, setMessage] = React.useState('');
+  const handleMessageChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setMessage(event.target.value);
+  };
+
+  // SUCCESS AND FAILURE MESSAGES
+  const successSnackBarStyle = useSuccessSnackBarStyle();
+  const failureSnackBarStyle = useFailsureSnackBarStyle();
+  const [successOpen, setSuccessOpen] = React.useState(false);
+  const [failOpen, setFailOpen] = React.useState(false);
+  const handleClose = (event: React.SyntheticEvent | React.MouseEvent, reason?: string) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setSuccessOpen(false);
+    setFailOpen(false);
+  };
+
+  // FORM SUBMISSION
+  const resetInputs = () => {
+    setName('');
+    setEmail('');
+    setMessage('');
+  }
+
+  const sendSubmission = async (data: IFormInput) => {
+    return fetch(
+      "https://api.airtable.com/v0/appro4l8TqwcZvFsC/Contact%20Form",
+      {
+        method: "post",
+        headers: new Headers({
+          Authorization: "Bearer keyhfYL0UX5ysZchd",
```

Comment:
What account is this? Is this a public account or your personal account? We only want public Airtable credentials being used on the front end.

Reply:
I copied the credentials and airtable location from the current landing page. Should I set up a testing account as a placeholder?

Reply:
Yeah, that's actually something in the current landing page that we don't want to copy. If we're starting on Airtable integration, we'll want a public view / form submit account setup with only the necessary permissions.

Reply:
I removed the credentials. Do you mind if we set up Airtable in a separate issue/PR?